static/posit/specialized/posit_16_2.cpp

```diff
@@ -12,9 +12,10 @@
 // second: enable posit arithmetic exceptions
 #define POSIT_THROW_ARITHMETIC_EXCEPTION 1
 #include <universal/number/posit/posit.hpp>
+#include <universal/number/posito/posito.hpp>
 #include <universal/verification/posit_test_suite.hpp>
 #include <universal/verification/posit_test_suite_mathlib.hpp>
-#include <universal/verification/posit_test_suite_randoms.hpp>
```

Comment:
Let's not modify the regression suite, but instead, let's add a new exploratory test so we can RCA this issue

Reply:
@Ravenwater I have a few issues with what you ask:
- How do you propose to do the type generalization in `posit_test_suite_randoms.hpp`? As it is now it's not straightforward to do, and I'm not sure what your plans are.
- To keep the `test_reporters.hpp` as-is, we'd need some sort of conversion between posit and posito. Where should that be added? I guestt it should suffice by copying the bits, but I might be wrong.
- Why not keep these extra tests in the `posit/specialized` folder? That's where the tests should be, not in some posito folder since we're not testing for posito. I'm saying this especially because these tests are passing. It also serves as a way of avoiding the issue we had at the beginning where the specialized posit<16,2> was behaving as a posit<16,1> and still passing these tests.

Reply:
@davidmallasen you are correct, we want to bring in posito<> as an oracle. 

Your mod got caught in the work-in-progress of unifying the regression API among all the number systems, which tripped over SFINAE failures. I am going to take your design and merge it into the new API and try to make it do double duty between marshalling through double and using an oracle.

Reply:
Perfect! Feel free to use my modifications in this PR in any way you are designing the new API. If at any point this PR is no longer relevant we can close it.